L1Trigger/Phase2L1GT/python/l1tGTAlgoBlockProducer_cff.py

```diff
@@ -1,6 +1,9 @@
 import FWCore.ParameterSet.Config as cms
 import re
 
+
+TYPE_STANDARD_PHYSICS = cms.vint32(1) # TODO
```

Comment:
Does this `TODO` need to be implemented?

Reply:
Trigger types are still open for discussion (see TDR). We'll decide on this at a later date what the actual trigger types should be.